lazy_tensor_core/test/cpp/test_aten_ltc_ts_tensor.cpp

```diff
@@ -2858,6 +2858,17 @@ TEST_F(AtenLtcTsTensorTest, TestEmpty) {
   });
 }
 
+TEST_F(AtenLtcTsTensorTest, TestZeroInPlace) {
+  torch::Tensor a = torch::zeros({2, 2}, torch::TensorOptions(torch::kFloat));
```

Comment:
Can we generate a non-zero initial tensor here? That will make sure we are really testing the in-place semantic here.

Reply:
Basically using the same initial tensor for lazy and non-lazy devices.

Reply:
I can move b to here. However, I'm not quite sure why that can test the in-place semantic and the current test can't.